h2o-algos/src/main/java/hex/schemas/StackedEnsembleV99.java

```diff
@@ -32,27 +33,27 @@
 
     
     // Metalearner algorithm
-    @API(level = API.Level.critical, direction = API.Direction.INOUT, gridable = true,
+    @API(level = API.Level.critical, direction = API.Direction.INOUT, 
             values = {"AUTO", "glm", "gbm", "drf", "deeplearning"},
             help = "Type of algorithm to use as the metalearner. " +
                     "Options include 'AUTO' (GLM with non negative weights; if validation_frame is present, a lambda search is performed), 'glm' (GLM with default parameters), 'gbm' (GBM with default parameters), " +
                     "'drf' (Random Forest with default parameters), or 'deeplearning' (Deep Learning with default parameters).")
     public StackedEnsembleModel.StackedEnsembleParameters.MetalearnerAlgorithm metalearner_algorithm;
 
     // For ensemble metalearner cross-validation
-    @API(level = API.Level.critical, direction = API.Direction.INOUT, gridable = true,
+    @API(level = API.Level.critical, direction = API.Direction.INOUT, 
             help = "Number of folds for K-fold cross-validation of the metalearner algorithm (0 to disable or >= 2).")
     public int metalearner_nfolds;
 
     // For ensemble metalearner cross-validation
-    @API(level = API.Level.secondary, direction = API.Direction.INOUT, gridable = true,
```

Comment:
Not sure that this is very useful for grid search, but in theory we could support it, so maybe we should leave it here?

Reply:
I was thinking to turn of `griddable` for all params since SE is not really supported in Grid yet.  Left this in by mistake.

Reply:
Same as above.